frontend/templates/product-list/sections/HeroSection.tsx

```diff
@@ -25,33 +23,22 @@ export interface HeroSectionProps {
 export function HeroSection({ productList }: HeroSectionProps) {
    const pagination = usePagination();
    const page = pagination.currentRefinement + 1;
-   const itemType = useDevicePartsItemType(productList);
-   const hasDescription =
-      productList.description != null &&
-      productList.description.length > 0 &&
-      page === 1 &&
-      !itemType;
-   const hasTagline =
-      productList.tagline != null &&
-      productList.tagline.length > 0 &&
-      page === 1 &&
-      !itemType;
-
-   const title = getProductListTitle(productList, itemType);
 
    return (
       <Flex direction="column">
          <HeroTitle>
-            {title}
+            {productList?.overrides?.title}
```

Comment:
I'm not sure why we went from `productList.` to `productList?.` here and in several other files. It seems like `productList` will always be defined in these contexts.